website/docs/operations/monitoring.mdx

```diff
@@ -130,3 +139,20 @@ Monitor Weave Gitops GO runtime metrics like Memory Usage, Memory Heap, Goroutin
 **Explorer**
 
 Monitor Explorer golden signals. More info [here](../../explorer/operations#monitoring)
+
+### Profiling
+
+Profiling can be useful during operation to help you to gain deeper understanding of the management cluster is behaving a runtime.
```

Comment:
`of the management cluster is behaving a runtime` should it be `of how the management cluster is behaving a runtime`?

`Go,profiling` - a space before the comma is missing.

Reply:
thanks, updated to make it a bit more clear.